.github/workflows/load-metrics.yml

```diff
@@ -45,7 +45,13 @@ jobs:
         run: |
           echo ${{ steps.ip.outputs.ipv4 }}
 
+      - name: "Set up Cloud SDK"
+        uses: "google-github-actions/setup-gcloud@v2"
+        with:
+          version: ">= 363.0.0"
+
       - name: Whitelist Github Action and Superset IPs
+        shell: bash -l {0}
```

Comment:
This was necessary for this step but not the later IP-clearing step? Weird... if you have any context on what was going on that necessitated this it might be nice to include as a comment.

Reply:
Good catch. It's not necessary, just an artifact of me attempting to fix this in 2 different ways. I removed it, the other changes are doing the work here.